docker/production/Dockerfile

```diff
@@ -15,33 +15,34 @@ EXPOSE 3000
 
 ENTRYPOINT ["/usr/src/app/docker/entrypoint-worker.sh"]
 
-# https://github.com/nodesource/distributions#installation-instructions
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
-
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-
-RUN apt-get update && apt-get install -y nodejs yarn ffmpeg imagemagick pdftk ghostscript sqlite3 rsync shared-mime-info --no-install-recommends && rm -rf /var/lib/apt/lists/*
-
-
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash \
- && apt-get update && apt-get install -y nodejs && rm -rf /var/lib/apt/lists/* \
- && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
- && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
- && apt-get update && apt-get install -y yarn && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ffmpeg imagemagick pdftk ghostscript sqlite3 rsync shared-mime-info && \
```

Comment:
Why did we install sqlite? Can it be removed?

Reply:
It is probably a remnant from the first dockerfiles where we used sqlite in development.

Reply:
So it can be removed.